src/rust/src/conversion/mod.rs

```diff
@@ -728,6 +728,19 @@ impl TryFrom<&str> for Wrap<AsofStrategy> {
     }
 }
 
+impl TryFrom<&str> for Wrap<RoundMode> {
+    type Error = savvy::Error;
+
+    fn try_from(round_mode: &str) -> Result<Self, Self::Error> {
+        let parsed = match round_mode {
+            "half_to_even" => RoundMode::HalfToEven,
+            "half_away_from_zero" => RoundMode::HalfAwayFromZero,
+            _ => return Err("unreachable".to_string().into()),
```

Comment:
```suggestion
            _ => return Err("unreachable".to_string()),
```

Reply:
This change broke the build.

Reply:
Shouldn't the output be `Result<Self, String>` like the rest of conversion functions? That's why it broke the build.

Reply:
Error types are not unified. I think we need to unify everything to `savvy::Error` eventually.

Reply:
Ok but then I'd say let's do all at once in another PR then and use the same output types as before here

Reply:
Again, there are already functions that use `savvy::Error`. There is no uniformity in which one is used.